src/partials/head-scripts.hbs

```diff
@@ -7,9 +7,8 @@
     new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
     j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
     'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-PSHTV2W');</script>
+    })(window,document,'script','dataLayer','GTM-5FSG7Q');</script>
     <!-- End Google Tag Manager -->
-
```

Comment:
I don't know whether removing this blank line affects anything. It's probably fine?